deploy.cmd

```diff
@@ -65,6 +65,13 @@ IF /I "%IN_PLACE_DEPLOYMENT%" NEQ "1" (
   IF !ERRORLEVEL! NEQ 0 goto error
 )
 
+:: 2. Check if WebConfig File exists
+IF EXIST %DEPLOYMENT_SOURCE%/web.config (
+  cp %DEPLOYMENT_SOURCE%/web.config %DEPLOYMENT_TARGET%/web.config
```

Comment:
That's weird. I'm running the `deploy.cmd` both `getruby.cmd` from this repository.
And the `web.config` file isn't created (generated or copied) from the repository to the `_site` folder. Either being copied to `wwwroot`.

I think this is a good workaroung. Can also do double checks.